sound/soc/sof/ipc.c

```diff
@@ -784,6 +784,34 @@ int snd_sof_ipc_set_get_comp_data(struct snd_sof_control *scontrol,
 }
 EXPORT_SYMBOL(snd_sof_ipc_set_get_comp_data);
 
+/* Helper of sending IPC to enable the DSP cores with cores_mask */
+int snd_sof_ipc_core_enable(struct snd_sof_dev *sdev, u32 core_mask)
```

Comment:
what is the purpose of this function? is it just for enabling or meant to be used for disabling as well? If it is for both, why do we only check enabled_cores_mask()? And how do you interpret this? lets say you want to enable core 2, you'd pass 0x4 but does that mean you'd turn off core 1 and core 0( possibly not because it is the primary core).

Reply:
@ranj063 can we set coreX other than core0 for tgl or adl+ ?

Reply:
> @ranj063 can we set coreX other than core0 for tgl or adl+ ?

For FW, yes. For driver, we just do nothing for non-0 cores.